recipe/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
   run:
     - memory_profiler
     - python
-    - spyder >=3
+    - spyder >=4
```

Comment:
I think we should limit this to

    - spyder >=4,<5

@jitseniesen, what do you think?

Reply:
I like your idea, I will do that in the future

Reply:
Ok, thanks!